packages/toolbox/package.json

```diff
@@ -13,7 +13,9 @@
   "dependencies": {
     "@mainframe/config": "*",
     "execa": "^1.0.0",
-    "fs-extra": "^7.0.0"
+    "fs-extra": "^7.0.0",
+    "keytar": "^4.3.0",
```

Comment:
I think `keytar` should be added as a dependency to the launcher, not this package.
The main reason is that it requires compiling native code, and this is done by `electron-builder` in the launcher, but if the `keytar` dependency is in this package, it won't be rebuilt.

Reply:
can we discuss this one further on Monday morning?

Reply:
Yes let me know when you're available to chat if you need